### PR TITLE
[PaddleDetection] update GETTING_STARTED.md

### PR DESCRIPTION
**Pull request by yghstill**
update some important information

### CODE REVIEW

File: PaddleCV/PaddleDetection/docs/GETTING_STARTED.md [hunk 1]
```diff
@@ -6,23 +6,66 @@ instructions](INSTALL.md).
...
+```bash
+export CPU_NUM=8
 python tools/train.py -c configs/faster_rcnn_r50_1x.yml
 ```
 
+##### Optional arguments
+
+- `-r`: Checkpoint path for resuming training. Such as: `-r output/faster_rcnn_r50_1x/10000`
+- `--eval`: Whether to perform evaluation in training, default is `False`
+- `-f`: If perform evaluation in train, this edits evaluation directory, default is current directory.
```
Comment (jerrywgz) [inline PaddleCV/PaddleDetection/docs/GETTING_STARTED.md]: train->training

File: PaddleCV/PaddleDetection/docs/GETTING_STARTED.md [hunk 2]
```diff
@@ -6,23 +6,66 @@ instructions](INSTALL.md).
...
+- `-d`: Dataset path, same as `dataset_dir` of configs. Such as: `-d dataset/coco`
+- `-o`: Set configuration options in config file. Such as: `-o weights=output/faster_rcnn_r50_1x/model_final`
+
+
+##### Examples
+
+- Perform evaluation in train
```
Comment (jerrywgz) [inline PaddleCV/PaddleDetection/docs/GETTING_STARTED.md]: training

File: PaddleCV/PaddleDetection/docs/GETTING_STARTED.md [hunk 3]
```diff
@@ -31,20 +74,49 @@ python tools/train.py -c configs/faster_rcnn_r50_1x.yml
...
+                        -d dataset/coco
+```
+
+- Evaluation with json
+```bash
+export CUDA_VISIBLE_DEVICES=0
+export PYTHONPATH=$PYTHONPATH:.
+# or run on CPU with:
+# export CPU_NUM=1
+python tools/eval.py -c configs/faster_rcnn_r50_1x.yml --json_eval
```
Comment (jerrywgz) [inline PaddleCV/PaddleDetection/docs/GETTING_STARTED.md]: json_eval后面给个路径？

File: PaddleCV/PaddleDetection/docs/GETTING_STARTED.md [hunk 4]
```diff
@@ -57,30 +129,54 @@ moment, but it is a planned feature
...
+# export CPU_NUM=1
+python tools/infer.py -c configs/faster_rcnn_r50_1x.yml \
+                      --infer_img=demo/000000570688.jpg \
+                      --output_dir=infer_output/ \
+                      --draw_threshold=0.5
+```
 The visualization files are saved in `output` by default, to specify a different
-path, simply add a `--output_dir=` flag.
+path, simply add a `--output_dir=` flag.         
+`--draw_threshold` is optional arguments. Different thresholds will produce different results depending on the calculation of [NMS](https://ieeexplore.ieee.org/document/1699659)
```
Comment (jerrywgz) [inline PaddleCV/PaddleDetection/docs/GETTING_STARTED.md]: an optional argument

Comment (jerrywgz) [reply PaddleCV/PaddleDetection/docs/GETTING_STARTED.md]: 给下默认值

File: PaddleCV/PaddleDetection/docs/GETTING_STARTED.md [hunk 5]
```diff
@@ -31,20 +74,49 @@ python tools/train.py -c configs/faster_rcnn_r50_1x.yml
...
 # export CPU_NUM=1
 python tools/eval.py -c configs/faster_rcnn_r50_1x.yml
 ```
 
+#### Optional arguments
+
+- `-d`: Dataset path, same as dataset_dir of configs. Such as: `-d dataset/coco`
+- `-f`: Evaluation directory, default is current directory.
+- `-o`: Set configuration options in config file. Such as: `-o weights=output/faster_rcnn_r50_1x/model_final`
+- `--json_eval`: Whether to eval with already exists bbox.json or mask.json. Default is `False`.
```
Comment (jerrywgz) [inline PaddleCV/PaddleDetection/docs/GETTING_STARTED.md]: json_eval 应该是表示为路径吧，如果是false表示不eval已有json

File: PaddleCV/PaddleDetection/docs/GETTING_STARTED.md [hunk 6]
```diff
@@ -91,8 +187,13 @@ Save inference model by set `--save_inference_model`, which can be loaded by Pad
 
 **Q:**  Why do I get `NaN` loss values during single GPU training? </br>
 **A:**  The default learning rate is tuned to multi-GPU training (8x GPUs), it must
-be adapted for single GPU training accordingly (e.g., divide by 8).
-
+be adapted for single GPU training accordingly (e.g., divide by 8).       
+The calculation rules are as follows，they are equivalent: </br>     
+| GPU number  | Learning rate  | Max_iters | Milestones       |
```
Comment (jerrywgz) [inline PaddleCV/PaddleDetection/docs/GETTING_STARTED.md:199]: 表头前换行

Comment (yghstill) [reply PaddleCV/PaddleDetection/docs/GETTING_STARTED.md:199]: DONE

File: PaddleCV/PaddleDetection/docs/GETTING_STARTED.md [hunk 7]
```diff
@@ -6,23 +6,66 @@ instructions](INSTALL.md).
...
 ```bash
 export CUDA_VISIBLE_DEVICES=0,1,2,3,4,5,6,7
+export PYTHONPATH=$PYTHONPATH:.
+python tools/train.py -c configs/faster_rcnn_r50_1x.yml
+```
+
+#### CPU Training
+
+```bash
+export CPU_NUM=8
```
Comment (jerrywgz) [inline PaddleCV/PaddleDetection/docs/GETTING_STARTED.md:29]: 这里也要设置export PYTHONPATH=$PYTHONPATH:.吧

File: PaddleCV/PaddleDetection/docs/GETTING_STARTED.md [hunk 8]
```diff
@@ -6,23 +6,66 @@ instructions](INSTALL.md).
...
+python -u tools/train.py -c configs/faster_rcnn_r50_1x.yml \
+                         -o weights=output/faster_rcnn_r50_1x/model_final \
+                         -d dataset/coco
+```
+
+
+##### NOTES
+
+- `CUDA_VISIBLE_DEVICES` Can specify different gpu numbers. Such as: `export CUDA_VISIBLE_DEVICES=0,1,2,3`. GPU calculation rules can refer [FAQ](#faq)
 - Datasets is stored in `dataset/coco` by default (configurable).
```
Comment (jerrywgz) [inline PaddleCV/PaddleDetection/docs/GETTING_STARTED.md]: Datasets-> Dataset

File: PaddleCV/PaddleDetection/docs/GETTING_STARTED.md [hunk 9]
```diff
@@ -6,23 +6,66 @@ instructions](INSTALL.md).
...
 - Datasets will be downloaded automatically and cached in `~/.cache/paddle/dataset` if not be found locally.
```
Comment (jerrywgz) [inline PaddleCV/PaddleDetection/docs/GETTING_STARTED.md]: same as above

File: PaddleCV/PaddleDetection/docs/GETTING_STARTED.md [hunk 10]
```diff
@@ -31,20 +74,53 @@ python tools/train.py -c configs/faster_rcnn_r50_1x.yml
 - RCNN models training on CPU is not supported on PaddlePaddle<=1.5.1 and will be fixed on later version.
```
Comment (jerrywgz) [inline PaddleCV/PaddleDetection/docs/GETTING_STARTED.md:75]: Model checkpoints is saved->Model checkpoints are saved

File: PaddleCV/PaddleDetection/docs/GETTING_STARTED.md [hunk 11]
```diff
@@ -31,20 +74,53 @@ python tools/train.py -c configs/faster_rcnn_r50_1x.yml
...
 # export CPU_NUM=1
 python tools/eval.py -c configs/faster_rcnn_r50_1x.yml
 ```
 
+#### Optional arguments
+
+- `-d`: Dataset path, same as dataset_dir of configs. Such as: `-d dataset/coco`
+- `-f`: Evaluation directory, default is current directory.
+- `-o`: Set configuration options in config file. Such as: `-o weights=output/faster_rcnn_r50_1x/model_final`
+- `--json_eval`: Whether to eval with already exists bbox.json or mask.json. Default is `False`. Json file directory assigned by `-f` argument.
```
Comment (jerrywgz) [inline PaddleCV/PaddleDetection/docs/GETTING_STARTED.md]: exists-> existed

Comment (yghstill) [reply PaddleCV/PaddleDetection/docs/GETTING_STARTED.md]: DONE

File: PaddleCV/PaddleDetection/docs/GETTING_STARTED.md (hunk 10 above)
Comment (yghstill) [reply PaddleCV/PaddleDetection/docs/GETTING_STARTED.md:75]: DONE

File: PaddleCV/PaddleDetection/docs/GETTING_STARTED.md (hunk 9 above)
Comment (yghstill) [reply PaddleCV/PaddleDetection/docs/GETTING_STARTED.md]: DONE

File: PaddleCV/PaddleDetection/docs/GETTING_STARTED.md (hunk 8 above)
Comment (yghstill) [reply PaddleCV/PaddleDetection/docs/GETTING_STARTED.md]: DONE

File: PaddleCV/PaddleDetection/docs/GETTING_STARTED.md (hunk 7 above)
Comment (yghstill) [reply PaddleCV/PaddleDetection/docs/GETTING_STARTED.md:29]: DONE

**Review by jerrywgz — APPROVED** [review]
LGTM

**Pull request merged**